### DISCUSSION TITLE
[Unstable] A collection of module problems

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14541] [Bug reports]
### What happened?

Z fighting in MedicalModule_G1: The Nexpharma sign on the ground has the same sprite depth as the separatist decoration, which leads to flickering

There is a cabinet behind some glass in MedicalModule_G2 that is interactable. It's clearly meant to be in a different room and has no loot

![image](https://github.com/user-attachments/assets/c67751f4-e0d0-4bac-8a04-72066eab55b4)

There is an unpowered light in the Admin_G4 module

![image](https://github.com/user-attachments/assets/4ba39f9c-74c6-4381-bd06-e64cd72f795d)

There is a pump inside the MineModule_4 that is always set to -100, which means it starts smoking which looks weird. It should have a water detector setup like all the other pumps instead.

**Comment by ubertpendragon** [comment]
![image](https://github.com/user-attachments/assets/21c8a868-56b0-4039-8fc2-6b2b29a07a23)

There's no background behind the glass

Are the offices underwater?

**Comment by Regalis11** [comment]
These have now been addressed in our private repo, the fixes are coming up in the next unstable build.
